### PR TITLE
Backend Changes - Forgot Password

### PR DESCRIPTION
**Pull request by kaustubha-m**

### CODE REVIEW

File: src/com/bluewhale/bus/dao/UserVerficationDao.java [hunk 1]
```diff
@@ -1,9 +1,15 @@
...
 
 public interface UserVerficationDao extends BaseDao {
+
 	String create(String username);
 
 	boolean verify(String username, String otp);
 
 	void delete(String username);
+
+	boolean isUserAlreadyCreated(String username);
```
Comment (codewnw) [inline src/com/bluewhale/bus/dao/UserVerficationDao.java]: Change method name from **isUserAlreadyCreated** to **isExistingUser**

File: src/com/bluewhale/bus/servlet/ForgotPassword.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import com.bluewhale.bus.service.UserVerficationService;
+import com.bluewhale.bus.service.UserVerficationServiceImpl;
+
+@WebServlet("/forgot")
+public class ForgotPassword extends HttpServlet {
```
Comment (codewnw) [inline src/com/bluewhale/bus/servlet/ForgotPassword.java]: Can we try to accommodate this code in `LoginServlet`?

File: src/com/bluewhale/bus/servlet/ResetPassword.java [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import com.bluewhale.bus.service.UserVerficationService;
+import com.bluewhale.bus.service.UserVerficationServiceImpl;
+
+@WebServlet("/resetpassword")
+public class ResetPassword extends HttpServlet {
```
Comment (codewnw) [inline src/com/bluewhale/bus/servlet/ResetPassword.java]: Can we try to accommodate this code in `LoginServlet`?

**Comment by kaustubha-m** [comment]
Check-in Done - As per above comments

**Review by codewnw — APPROVED** [review]

**Pull request merged**